### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by romulogcerqueira**
* Add default values to start_angle, end_angle, step_angle, beam_width and beam_height properties
* Minor improvements
* Add unitary tests

### CODE REVIEW

File: tasks/MultibeamSonarTask.cpp [hunk 1]
```diff
@@ -8,17 +8,21 @@ using namespace base::samples::frame;
 
 MultibeamSonarTask::MultibeamSonarTask(std::string const& name) :
 		MultibeamSonarTaskBase(name) {
+	_beam_width.set(base::Angle::fromRad(120.0 * M_PI / 180.0));
```
Comment (doudou) [inline tasks/MultibeamSonarTask.cpp]: There are methods in base::Angle to initialize from degrees

Comment (doudou) [inline tasks/MultibeamSonarTask.cpp]: Also, better initialize attributes in the initializer list

```
   : MultibeamSonarTaskBase(name)
   , _beam_width(base::Angle::fromRad(120.0 * M_PI / 180.0))
   , ...
```

Comment (romulogcerqueira) [inline tasks/MultibeamSonarTask.cpp]: > There are methods in base::Angle to initialize from degrees

No problem, I can get this way. Done!

Comment (romulogcerqueira) [inline tasks/MultibeamSonarTask.cpp]: >Also, better initialize attributes in the initializer list

I tried to follow this way, however the compiler didn't recognize the properties in the initializers list.

Comment (romulogcerqueira) [inline tasks/MultibeamSonarTask.cpp]: The compiler output:
<pre>
    [ 78%] Building CXX object tasks/CMakeFiles/sonar_imaging_autoranging-tasks-gnulinux.dir/Task.cpp.o
    /home/romulo/dev.bir/drivers/orogen/sonar_imaging_autoranging/tasks/Task.cpp: In constructor ‘sonar_imaging_autoranging::Task::Task(const string&)’:
    /home/romulo/dev.bir/drivers/orogen/sonar_imaging_autoranging/tasks/Task.cpp:10:7: error: class ‘sonar_imaging_autoranging::Task’ does not have any field named ‘_angle_tilt’
         , _angle_tilt(base::Angle::fromRad(M_PI))
           ^

</pre>

Comment (doudou) [inline tasks/MultibeamSonarTask.cpp]: Sorry, my bad. These are part of the base class, and so cannot be initialized in the initializer list.

**Pull request merged**